accessibility/tools.md

```diff
@@ -59,16 +59,18 @@ On mobile, VoiceOver is built-in on iOS devices, and TalkBack is usually built-i
 
 ## Other accessibility tools
 
-### Accessibility testing tools in the browser
+### Automated accessibility testing tools in the browser
 
 - Squizlabs also have an [HMTML_Codesniffer bookmarklet](https://squizlabs.github.io/HTML_CodeSniffer/) for quick tests. 
 - The Axe engine comes as a [browser extension](https://www.deque.com/axe/browser-extensions/) for Chrome, Firefox, and Edge, and as an [NPM module](https://github.com/dequelabs/axe-core) that you can integrate with your build, like Pa11y. 
 - WebAim's [WAVE extension](https://wave.webaim.org/extension/) for Chrome and Firefox evaluates accessibility in place on the page. 
-- The [ARC Toolkit](https://www.paciellogroup.com/toolkit/) by the Paciello Group is a useful extension for Chrome that (among other things) can detect semantic/structural issues, and problematic use of ARIA. 
+- The [ARC Toolkit](https://www.paciellogroup.com/toolkit/) by TPGi is a useful extension for Chrome that (among other things) can detect semantic/structural issues, and problematic use of ARIA. 
```

Comment:
`https://www.paciellogroup.com/toolkit/` gets redirected to `https://www.tpgi.com/arc-platform/arc-toolkit/`. Shall we use that instead?

Reply:
Yuss thanks! Pushed a new commit 👍